debian/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+# output every command that modifies files on the build system.
+export DH_VERBOSE = 1
+export DH_OPTIONS=-v
+
+export PYBUILD_NAME=bitmessage
+
+%:
+	dh $@ --with python2 --buildsystem=pybuild
```

Comment:
Did you try --sourcedirectory=.. here?

Reply:
![image](https://user-images.githubusercontent.com/33522186/95771608-fd798180-0cd8-11eb-81c4-784291ff24cf.png)
I have done some changes, please look

Reply:
add the argument to the existing command, don't add a new command

Reply:
ok now please review it
![image](https://user-images.githubusercontent.com/33522186/95777954-c27d4b00-0ce4-11eb-92d9-57ba5e65daed.png)

Reply:
@navjotcis I meant after moving the stuff to the packages dir.

Reply:
Well, I checked: --sourcedirectory=.. not works.

Reply:
yes it is failing the build